### ISSUE TITLE
Request: "create branch" contextual menu option

### ISSUE THREAD

**Issue by onecrayon** [sublimehq/sublime_merge#299]
**Problem description**

Currently, the only way to create a new branch is to choose Respository -> Create Branch or use the command palette.

**Preferred solution**

It would be very helpful if you could right click a local branch and choose "Create Branch" from the context menu (this would also slightly speed up the workflow for creating a new branch when you're on a different branch than your desired root branch).

**Comment by wbond** [comment]
So to clarify: the request is to add a context menu entry to branches in the Location Bar that allows creating a new branch from the selected branch.

**Comment by onecrayon** [comment]
> So to clarify: the request is to add a context menu entry to branches in the Location Bar that allows creating a new branch from the selected branch.

Yes, exactly. Basically just an alias for "Create branch at commit" that works on the latest commit for a given branch.
